actions/java-gradle-build-jib/action.yaml

```diff
@@ -89,6 +89,8 @@ runs:
         tags: |
           event=tag,type=semver,pattern={{version}}
           event=push,type=raw,value=pipeline-${{ github.run_id }}-git-{{sha}},enable=${{ github.ref_type != 'tag' }}
+          event=pr,type=ref
```

Comment:
As far as I understood, this does not work because the way we inject it into jib, only one tag can be used. I therefore suggest to change our jib usage a bit. Always use the pipeline tag as parameter for `jibImage.tag` and pass additional tags created by metadata action to `jib.to.tags` (git tag, latest, pr)